Allow tuning red color sensitivity of infra-red distance sensor (#3077)
cyberbotics/webots: docs/reference/distancesensor.md

```
DistanceSensor {
  MFVec3f  lookupTable              [ 0 0 0, 0.1 1000 0 ] # lookup table
  SFString type                     "generic"             # {"generic", "infra-red", "sonar", "laser"}
  SFInt32  numberOfRays             1                     # [1, inf)
  SFFloat  aperture                 1.5708                # [0, 2*pi]
  SFFloat  gaussianWidth            1                     # [0, inf)
  SFFloat  resolution               -1                    # {-1, [0, inf)}

>   SFFloat  redColorSensitivity      1                     # [0, 1]

Reviewer: That seems more correct:
```suggestion
  SFFloat  redColorSensitivity      1                     # [0, inf)
```

Author: Actually even value bigger than 1 will work, it will increase the red color sensitivity of the sensor above the original Webots behavior. But we can go with 0-1, no problem. I even mention it in the WRL file. Maybe I should add a validation then?

Setting this field to -1 (default) means that the sensor has an 'infinite' resolution (it can measure any infinitesimal change).
This field accepts any value in the interval (0.0, inf).

- `redColorSensitivity`: red color sensitivity factor.
This allows to tune (or even disable) red color sensitivity for infra-red distance sensor type.

> Value of 1 means ordinary Webots behavior.

Reviewer: ```suggestion
A value of 1 corresponds to the default behavior.
```

Values greater that 1 increase the red color sensitivity and values lower than 1 decrease it.

> Value of 0 disables the red color sensitivity completely.

Reviewer: ```suggestion
A value of 0 disables the effect of the red color completely.
```
